Common/Retranslator/src/Retranslator.hpp

```diff
@@ -0,0 +1,33 @@
+#include <memory>
+#include "AST.hpp"
+#include <map>
+#include "Shape.hpp"
+#include "Node.hpp"
+#include "Line.hpp"
+#include "Circle.hpp"
+#include "Diamond.hpp"
+#include "Reactangle.hpp"
+#include "Style.hpp"
+
+using namespace AST;
+class Retranslator{
+public:
+    static Retranslator& getInstance(){
```

Comment:
[minor] Сложно как-то, можно же просто
```suggestion
    static Retranslator& getInstance(){
        static Retranslator retranslator;
        return retranslator;
    };
```